packages/components/src/internal/components/auditlog/utils.ts

```diff
@@ -144,3 +146,27 @@ export function getTimelineEntityUrl(d: Record<string, any>): AppURL {
 
     return url;
 }
+
+export function getAuditDetailMap(
+    data: Record<string, string>,
+    inheritedFields?: string[]
+): OrderedMap<string, string> {
+    if (inheritedFields?.length > 0) {
+        // sort fields.newData so that inherited fields are at the bottom
+        const inheritedFieldsLc = inheritedFields.map(f => f.toLowerCase());
+        const newData = data ?? {};
+        const sortedNewData = {} as any;
+        const last = {};
+        for (const field of Object.keys(newData)) {
+            if (!newData[field] && inheritedFieldsLc.indexOf(QueryKey.encodePart(field).toLowerCase()) > -1) {
```

Comment:
as per my other comment, I assume that with the `QueryKey.encodePart()` call here that the `inheritedFields` are fieldKeys. If the data object is keyed by field names, can we just have `inheritedFields` be an array of field names so that we don't have to do that `QueryKey.encodePart()`?